db/models/index.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const fs        = require("fs");
+const path      = require("path");
+const Sequelize = require("sequelize");
+const basename  = path.basename(__filename);
+const env       = process.env.NODE_ENV || "development"; // eslint-disable-line no-process-env
+const config    = require(__dirname + "/../config/config.json")[env];
+const db        = {};
+
+let sequelize;
+
+if (config.use_env_constiable) {
```

Comment:
While struggling to figure out if "constiable" is a word, I think this is the unholy union of a search-replace `var` with `const`, and this should be `config.use_env_variable`.